openstack/octobus/templates/seeds.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: "openstack.stable.sap.cc/v1"
+kind: "OpenstackSeed"
+metadata:
+  name: octobus-seed
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  requires:
+  - monsoon3/domain-ccadmin-seed
+
+  domains:
+  - name: ccadmin
+    projects:    
+    - name: octobus
+      description: 'data collection, monitoring, analytics, event processing and storage infrastructure https://wiki.wdf.sap.corp/wiki/x/mqygd'
+      networks:
+      - name: default
+        subnets:
+         - name: octobus_private
+           cidr: 10.180.0.0/16
+           ip_version: 4
+      # Disabled because configuring subnets (instead of the parent network) during the seeding is not yet implemented
```

Comment:
You can now try with:
      routers:
      - name: default
        external_gateway_info:
          network: FloatingIP-external-ccadmin-monitoring@ccadmin-net-infra@ccadmin
          external_fixed_ips:
          - subnet: FloatingIP-sap-ccadmin-monitoring-01@ccadmin-net-infra@ccadmin
        interfaces:
        - subnet: octobus_private